content/_data/changelogs/weekly.yml

```diff
@@ -1562,6 +1562,55 @@
     # pull: 3153 too minor: Grammar fixes
     # pull: 3151 internal: Jenkinsfile
     # pull: 3158 internal: disable flaky test
+- version: "2.93"
+  date: 2017-12-03
+  changes:
+    - type: rfe
+      pull: 3135
+      message: >
+        Use Java NIO to read and write Unix file permissions by default.
+        The previous behavior can be restored by setting the Java system property <code>hudson.Util.useNativeChmodAndMode</code> to <code>true</code>.
+      references:
+        - issue: 36088
+        - url: https://wiki.jenkins-ci.org/display/JENKINS/Features+controlled+by+system+properties
+          title: Jenkins features controlled by system properties
+    - type: rfe
+      pull: 3167
+      message: >
+        Better handling of certain unreproducible XML file load/save errors.
+    - type: rfe
+      issue: 47429
+      pull: 3150
+      message: >
+        Improve user lookup performance, for example from Git changelog calculation.
+    - type: rfe
+      issue: 48348
+      pull: 3157
+      message: >
+        Reduce memory usage when scheduling pipelines on big clusters.
+    - type: rfe
+      issue: 34855
+      pull: 2548
+      message: >
+        Use atomic file moves if available on the underlying file system from <code>AtomicFileWriter</code>.
+    - type: bug
+      pull: 3164
+      issue: 48080
+      message: >
+        Prevent setup wizard from hanging when the two provided passwords differ, instead show a validation error.
+    - type: rfe
+      pull: 3148
+      message: >
+        Developer: Add <code>ItemGroup#allItems</code> and similar default methods to <code>ItemGroup</code>.
+    - type: rfe
+      pull: 3142
+      message: >
+        Developer: Add default implementations of deprecated methods to <code>BuildableItem</code> and <code>Item</code> so they don't need to be implemented.
```

Comment:
don't => do not ?
I am not sure how it will be rendered

Reply:
Properly, we had e.g. https://jenkins-ci.org/changelog/#v2.64 with https://github.com/daniel-beck/jenkins.io/blob/e63c571345943b172c4dab33bbce4fea94c078b8/content/_data/changelogs/weekly.yml#L674 and it was no problem.